apps/point-of-sale/src/components/Cart/TransactionHistory/TransactionHistoryComponent.vue

```diff
@@ -8,7 +8,7 @@
 */
 <template>
   <div class="flex-column h-full gap-2 mt-2 overflow-y-auto">
-    <h class="font-medium text-xl">Recent Purchases:</h>
+    <h1 class="font-medium text-xl" v-if="transactions.length > 0">Recent Purchases:</h1>
```

Comment:
This is so "Recent Purchases:" is not rendered if the list of transactions is empty